### PR TITLE
Hivemind buffs(*)

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Hivemind admin pannel now lets you properly see mobs lists
Hivemind admin pannel now lets you teleport to each master node of an active hivemind
Hivemind admin pannel now lets you add (and thus subtract) evo points to active hiveminds
Hivemind borrowing has been tweaked: By default hivemind will borrow only with non-maints borrows unless otherwise set.
Fixes mechiver null issue leading to endless converting by misstake

### CODE REVIEW

File: code/game/gamemodes/events/hivemind_controller.dm [hunk 1]
```diff
@@ -133,6 +149,11 @@ GLOBAL_VAR_INIT(hivemind_panel, new /datum/hivemind_panel)
 	if(href_list["area_list_interact"])
 		area_list_interact()
 
+	if(href_list["give_points"])
+		var/cap = input(usr, "How many points to give (Negitive numbers subtract)", "Hivemind Pitty Fund") as null|num
```
Comment (Smokingsquirrel) [inline code/game/gamemodes/events/hivemind_controller.dm:153]: „Negative“ and „pity“

Comment (Trilbyspaceclone) [inline code/game/gamemodes/events/hivemind_controller.dm:153]:
```suggestion
		var/cap = input(usr, "How many points to give (Negative numbers subtract)", "Hivemind Pity Fund") as null|num
```

File: code/game/gamemodes/events/hivemind_controller.dm [hunk 2]
```diff
@@ -1,6 +1,8 @@
 GLOBAL_LIST_INIT(hive_data_bool, list(
 	"maximum_existing_mobs"			= FALSE, // Exact number set separately
-	"spread_trough_burrows"			= FALSE,
+	"spread_trough_burrows"			= TRUE,
```
Comment (Trilbyspaceclone) [inline code/game/gamemodes/events/hivemind_controller.dm:3]:
```suggestion
	"spread_trough_burrows"			= FALSE,
```

File: code/game/gamemodes/events/hivemind_controller.dm [hunk 3]
```diff
@@ -95,10 +107,14 @@ GLOBAL_VAR_INIT(hivemind_panel, new /datum/hivemind_panel)
...
-	data += "<br>Spread trough burrows: [GLOB.hive_data_bool["spread_trough_burrows"] ? "Enabled" : "Disabled"] \
+	data += "<br><br>(Note: If enabled spread through burrows is also enabled)\
+	<br>Spread through maints burrows: [GLOB.hive_data_bool["spread_maints_burrows"] ? "Enabled" : "Disabled"]\
+	<a href='?src=\ref[src];toggle_maint_burrow=1'>\[TOGGLE\]</a>"
+
+	data += "<br>Spread through normal burrows (I.E borrows in hallways and departments.): [GLOB.hive_data_bool["spread_trough_burrows"] ? "Enabled" : "Disabled"] \
 	<a href='?src=\ref[src];toggle_burrow=1'>\[TOGGLE\]</a>"
 
-	data += "<br>Spread on z level below: [GLOB.hive_data_bool["spread_on_lower_z_level"] ? "Enabled" : "Disabled"] \
+	data += "<br<br>Spread on z level below: [GLOB.hive_data_bool["spread_on_lower_z_level"] ? "Enabled" : "Disabled"] \
```
Comment (Trilbyspaceclone) [inline code/game/gamemodes/events/hivemind_controller.dm:117]:
```suggestion
	data += "<br><br>Spread on z level below: [GLOB.hive_data_bool["spread_on_lower_z_level"] ? "Enabled" : "Disabled"] \
```

**Pull request merged**